### ISSUE TITLE
Accessibility of the 404 page

### ISSUE THREAD

**Issue by bl-aire** [jupyterhub/jupyterhub#4270]
### Details from report:

Based on evaluation using the WAVE accessibility tool, the following are responsible for accessibility issues on the 404 page:

The errors on this page are the same with those on the previous pages evaluated(Home, Login and Token) and can be fixed with the same solutions discussed earlier:

- **Language missing or invalid**
- **Skipped heading level**
- **Possible heading**
- **Noscript element**

### Report:

<img width="1280" alt="Screenshot 2022-12-14 at 11 59 47" src="https://user-images.githubusercontent.com/84882370/207596167-2ca5e07b-81f7-43ab-8dca-e52d9e771812.png">

@minrk

**Comment by minrk** [comment]
Great! Do you want to make a PR for the general issues (language, noscript) before evaluating more pages?

I don't quite know what the issue is with noscript.

**Comment by bl-aire** [comment]
> Great! Do you want to make a PR for the general issues (language, noscript) before evaluating more pages?

Yes. That would prevent unnecessary repetition in every issue.

> I don't quite know what the issue is with noscript.

I have been trying to understand it too. This is what I saw on WAVE:

> **Why It Matters**
Content within <noscript> is presented if JavaScript is disabled. Because nearly all users (including users of screen readers and other assistive technologies) have JavaScript enabled, <noscript> cannot be used to provide an accessible version of inaccessible scripted content.
**How to Fix It**
Ensure that scripted content is accessible. The <noscript> content will be presented to very few users, but must be accessible if used.

**Comment by minrk** [comment]
We don't use noscript to present anything about accessibility - it's a message that the pages mostly won't work, so I don't really understand the comments. It seems irrelevant.

**Comment by tpu01yzx** [comment]
I have the same problem. After login and redirect to the spamed jupyter_server page it shows 404 with URL _http://XX.XX.XX:280/user/yzx/tree?_.



> 404 : Not Found
> 
> You are requesting a page that does not exist!

The config is

> c = get_config()  #noqa
> 
> import sys
> 
> c.Authenticator.admin_users = {'root', 'yzx'}  # 管理员用户
> c.JupyterHub.admin_access = True
> c.PAMAuthenticator.open_sessions = False # 解决多用户同时登录问题。
> c.Spawner.args = ['--allow-root']  # 允许root用户使用
> c.LocalAuthenticator.create_system_users = True  # 允许创建其他用户
> c.Spawner.notebook_dir = '~'  # 设置工作目录
> #c.Spawner.default_url = '/lab'
> #c.JupyterHub.base_url='http://XXX.XXX.XXX:280'
> 
> c.JupyterHub.concurrent_spawn_limit = 128
> c.JupyterHub.extra_log_file = '/var/log/jupyterhub.log' # 指定额外的日志
> c.JupyterHub.pid_file='/srv/jupyterhub/jupyterhub.pid' # 指定pid文件位置
> c.JupyterHub.db_url='/srv/jupyterhub/jupyterhub.sqlite' # 指定数据库文件位置
> c.JupyterHub.cookie_secret_file='/srv/jupyterhub/jupyterhub_cookie_secret'  # 指定cookie_secret文件位置
> c.ConfigurableHTTPProxy.pid_file='/srv/jupyterhub/jupyterhub-proxy.pid' # 设置proxy.pid文件位置
> 
> 
> c.JupyterHub.services = [
>     {
>         'name': 'idle-culler',
>         'command': [sys.executable, '-m', 'jupyterhub_idle_culler', '--timeout=1800'],
>     }
> ]
> 
> c.JupyterHub.load_roles = [
>     {
>         "name": "list-and-cull",  # name the role
>         "services": [
>             "idle-culler",  # assign the service to this role
>         ],
>         "scopes": [
>             # declare what permissions the service should have
>             "list:users",  # list users
>             "read:users:activity",  # read user last-activity
>             "admin:servers",  # start/stop servers
>         ],
>     }
> ]
>
